pkg/autoscaler/autoscaler.go

```diff
@@ -257,16 +256,29 @@ func (a *Autoscaler) Scale(ctx context.Context, now time.Time) (int32, bool) {
 			if lastStat[stat.PodName].Time.Before(*stat.Time) {
 				lastStat[stat.PodName] = stat
 			}
+			// Update lastRequestTime if the current stat is newer and
+			// actually contains requests
+			if a.lastRequestTime.Before(*stat.Time) && stat.RequestCount > 0 {
+				a.lastRequestTime = *stat.Time
+			}
```

Comment:
Do we really need to bring the lastRequestTime back? Why?

Reply:
Oh, I thought we should use `ScaleToZeroGracePeriod`, but in fact the lastRequestTime 
 usually work together with `ScaleToZeroIdlePeriod`, I mixed the guard function of them. So if we had already decided to use the stable window, then I think it is nice to delete `lastRequestTime`.